pytest_fixtures/core/contenthosts.py

```diff
@@ -238,7 +245,8 @@ def module_flatpak_contenthost(request):
         "rhel_version": "9",
         "distro": "rhel",
         "no_containers": True,
-        "network": "ipv6" if settings.server.is_ipv6 else "ipv4",
+        # TODO(vsedmik): Check whether this is valid for dualstack scenaro
+        "network": "ipv6" if settings.server.network_type == NetworkType.IPV6 else "ipv4",
```

Comment:
Same proposal as for `module_container_contenthost`
```suggestion
        "network": settings.content_host.attributes.network_type,
```